hack/lib/etcd.sh

```diff
@@ -46,7 +46,7 @@ kube::etcd::validate() {
   fi
 
   # validate installed version is at least equal to minimum
-  version=$(etcd --version | tail -n +1 | head -n 1 | cut -d " " -f 3)
+  version=$(etcd --version | grep Version | tail -n +1 | head -n 1 | cut -d " " -f 3)
```

Comment:
@zhlhahaha `etcd --version | grep Version | cut -d " " -f 3` should be enough?

Reply:
No, it shows results as following:
`3.3.10
go1.10.4`
alternative script can be 
`etcd --version | grep "etcd Version" | cut -d " " -f 3`

Reply:
Which one you think better? @dims

Reply:
ah never mind then :)